src/main/asciidoc/_chapters/architecture.adoc

```diff
@@ -261,8 +261,10 @@ For region name, we only accept `byte[]` as the parameter type and it may be a f
 Information on non-Java clients and custom protocols is covered in <<external_apis>>
 
 [[client.masterregistry]]
-
 === Master Registry (new as of 2.3.0)
+Starting from 2.5.0, MasterRegistry is deprecated. RpcConnectionRegistry is the alternate. Please
```

Comment:
I think that the language can be a little stronger regarding the capabilities of RpcConnectionRegistry.

> Starting from 2.5.0, MasterRegistry is deprecated. It's functionality is completely superseded by the RpcConnectionRegistry. Please see ...